mne/viz/_brain/surface.py

```diff
@@ -121,9 +121,15 @@ def load_geometry(self):
             coords = coords[:, [1, 0, 2]]
             coords[:, 1] *= -1
         else:
-            coords, faces = read_surface(
-                path.join(self.data_path, 'surf',
-                          '%s.%s' % (self.hemi, self.surf)))
+            # allow ?h.pial.T1 if ?h.pial doesn't exist for instance
+            # end with '' for better file not found error
+            for img in ('', 'T1', 'T2', ''):
+                surf_fname = path.join(self.data_path, 'surf',
+                                       '.'.join((self.hemi, self.surf, img)
+                                                ).strip('.'))
```

Comment:
```suggestion
            for img in ('', '.T1', '.T2', ''):
                surf_fname = path.join(self.data_path, 'surf',
                                       '.'.join((self.hemi, self.surf, img)))
```
might be simpler to just put the `.` on the T1 or T2; WDYT?

Reply:
I'm happy either way